File: app/src/main/java/me/srikavin/quiz/model/AuthUser.java
package me.srikavin.quiz.model;

import com.google.gson.annotations.Expose;

public class AuthUser {
    @Expose
    private boolean auth;
    @Expose
    private String token;


    public String getToken() {
        return token;
    }

    public boolean isAuth() {
        return auth;
    }

}

File: app/src/main/java/me/srikavin/quiz/model/AuthUser.kt
package me.srikavin.quiz.model

import com.google.gson.annotations.Expose

/**
 * A container class representing the authentication token and the authentication state
 */
class AuthUser {
    /**
     * The username of the user
     */
    @Expose
    val username: String = ""
    /**
     * Represents whether the user is authenticated
     */
    @Expose
    val isAuth: Boolean = false
    /**
     * Contains the token that will be necessary for any further requests to the server
     */
    @Expose
    val token: String? = null
}
